src/entry.js

```diff
@@ -1,6 +1,6 @@
 import Clock from './Clock';
 
-import './Clock.css';
+import './Clock.less';
```

Comment:
That can't be like that - less is compiled to css on build for compatibility reasons. Not everybody uses less.